Sources/PasscodeKit/Views/KeypadView.swift

```diff
@@ -30,14 +35,42 @@ struct KeypadView: View {
                 NumberButton(value: .text("9"), text: $text)
             }
             HStack(spacing: configuration.hSpacing) {
-                NumberButton(value: .blank, text: $text)
-                    .hidden()
-                    .disabled(true)
+                if passcodeEnv.manager.passcode?.isBiometricsEnabled ?? false {
```

Comment:
Here we also need to do a check if biometrics are allowed and enabled. Simply enabled is not enough.